### ISSUE TITLE
question: recommended technique for "redirecting" in before hook

### ISSUE THREAD

**Issue by tony-kerz** [iron-meteor/iron-router#349]
i'm working with an example attempting to illustrate how to use `meteor-roles` for authorization in conjunction with `iron-router`.

the example checks for a user to be logged-in via before hook, and if not, "redirects" to a sign-in page, but it seems like it's not quite right, in that it calls `this.layout` (where `this` is a `RouteController` apparently), but i can't see that `RouteController` has a `layout` property:

https://github.com/alanning/meteor-roles/blob/v1.2.6/examples/iron-router/client/routing.js#L47

i see a `setLayout`, but i'm not sure if this will be considered after a call to `render`.

also, this is all followed by a `this.stop` and a `return`.

so basically i'm asking for a recommended technique for accomplishing the desired effect of intercepting and re-routing a call within a `before` hook.

the example re-routes because no user is logged in, but i could also see re-routing if the user is logged in but requesting a route for which they aren't authorized.
